interface/usergroup/addrbook_edit.php

```diff
@@ -528,5 +533,41 @@ function typeSelect(a) {
 
 </center>
 </form>
+
+<script>
+    
+    function validateEmail(emailField)
+    {
+        var email = $('#email_id').val();
+        var trusted_email = $('#trusted_email').val();
+        var reg = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+        if (reg.test(email) == false && email!='' ) 
+        {
+            alert('Invalid Email Address');
+            document.getElementById("form_save").disabled = true;  
+            return false;
+        }
+        if (reg.test(trusted_email) == false && trusted_email!='' ) 
+        {
+            alert('Invalid Trusted Email Address');
+            document.getElementById("form_save").disabled = true;
+            return false;
+        }
+        
+        else
```

Comment:
I agree...Indents should organize the code, and braces on extra lines just turn everything into a scrolling race, but stick to the file's convention, and make beautification a new project.
  We need standards published, and a general agreement (and a project plan) before making strong formatting arguments.